### ISSUE TITLE
Add support for `wordpress-muplugin` package type

### ISSUE THREAD

**Issue by benoitchantre** [inpsyde/wp-translation-downloader#13]
At the moment, the following packages types are supported:

- wordpress-core
- wordpress-plugin
- wordpress-theme
- library

It would be great to add `wordpress-muplugin` to that list.

**Comment by Chrico** [comment]
Good morning. Sorry for the late response (still in vacation :) ).

I guess it's fine to add `wordpress-muplugin`-type  to the supported list. Probably we shouldn't even restrict to types. Basically every custom composer type can provide translations.

Hm...the problem is, that there is no default WP.org REST Endpoint for getting translations from muplugins and we also have no API in WP Core to load translations for muplugins like theme/plugin translations. 
So we probably need to decide to put the translation files into a `muplugin`-folder in `languages` and document down, that you can load them like the type [`library`](https://github.com/inpsyde/wp-translation-downloader#support-for-composer-type-library). 🤔

**Comment by Chrico** [comment]
Hey ☕ 

With the latest implementation of https://github.com/inpsyde/wp-translation-downloader/pull/27 (not released yet), it is also possible to configure non-predefined types. You can use for example:

```json
{
	"languageRootDir": "public/languages/",
	"directories": {
		"types": {
			"wordpress-muplugin": "mu-plugins"
		}
	},
	"api": {
		"types": {
			"wordpress-muplugin": "https://some-custom-glotpress.com/api/%packageName%/"
		}
	}
}
```

which will download your translations for mu-plugins into `/{root-path}/public/languages/mu-plugins/`.

**Comment by Chrico** [comment]
This was released with the latest 2.1 version. Check out the changelog: https://github.com/inpsyde/wp-translation-downloader/releases/tag/2.1 :) 

@benoitchantre
